docs/tutorial/performance.md

```diff
@@ -432,3 +432,16 @@ As of writing this article, the popular choices include [Webpack][webpack],
 [parcel]: https://parceljs.org/
 [rollup]: https://rollupjs.org/
 [vscode-first-second]: https://www.youtube.com/watch?v=r0OeHRUCCb4
+
+
+### 8. Call `Menu.setApplicationMenu(null)` when you do not need a default menu
```

Comment:
This block should be above the large block of links above

Reply:
👍